packaging/post_build_script_linux.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -ex
+
+pip3 install auditwheel pkginfo
+
+for pkg in dist/torchdata*.whl; do
+    echo "PkgInfo of $pkg:"
+    pkginfo $pkg
+
+    auditwheel show $pkg
+    auditwheel repair $pkg --plat manylinux2014_x86_64
```

Comment:
```suggestion
    auditwheel repair $pkg --plat manylinux2014_x86_64
    auditwheel show wheelhouse/$pkg
```

Reply:
Hmmm, I checked [the order in the _build_test_upload workflow](https://github.com/pytorch/data/blob/main/.github/workflows/_build_test_upload.yml#L179-L180)

Reply:
Lmao, it means you just found a bug from the existing workflow.

Basically, I want to use `auditwheel show` to make sure the repaired wheels are manylinux compatible. However, calling `show` before `repair` won't give me those info.